test/Main.purs

```diff
@@ -12,9 +12,8 @@ main = do
   interface <- createConsoleInterface noCompletion
   setPrompt "> " interface
   prompt interface
-  setLineHandler interface $ \s ->
-    if s == "quit"
-       then close interface
-       else do
-        log $ "You typed: " <> s
-        prompt interface
+  setLineHandler (\s -> if s == "quit"
+                           then close interface
+                           else do
+                              log $ "You typed: " <> s
+                              prompt interface) interface
```

Comment:
This can use a bit less indentation with:

```suggestion
  interface # setLineHandler \s ->
    if s == "quit" then
      close interface
    else do
      log $ "You typed: " <> s
      prompt interface
```

Reply:
I updated the whitespace so it isn't like either of us! :P